Sources/AppStoreConnectCLI/Commands/TestFlight/Sync/TestFlightPullCommand.swift

```diff
@@ -25,6 +26,18 @@ struct TestFlightPullCommand: CommonParsableCommand {
     ) var outputPath: String
 
     func run() throws {
+        let service = try makeService()
+
+        // TODO: A new service function should be created to efficiently gather these models
+        let apps = try service.listApps(bundleIds: filterBundleIds)
+        let identifiers = apps.map { app in AppLookupIdentifier.appId(app.id) }
+        let testers = try service.listBetaTesters(filterIdentifiers: identifiers)
+        let groups = try service.listBetaGroups(filterIdentifiers: identifiers)
```

Comment:
We might need another request for requesting all the testers in the beta group

Reply:
From what I can see at the moment we have all the information required.
We have all the testers for all the specified apps, we have all the groups for the specified apps and the testers list what groups they are a part of.

But maybe it will be better to do it that way when we re-write the service function...

The testers only list the group names they are a part of and they could be duplicated across apps

Reply:
yes, that's what I am worried about, tester only has an of group names instead of `ID`.

Reply:
Yeah I think you're right. I can take a look for this PR